ksql-examples/pom.xml

```diff
@@ -38,6 +38,12 @@
             <version>${arg.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>monitoring-interceptors</artifactId>
+            <version>${project.version}</version>
```

Comment:
Looks ok. Looking at other entires, none seem to explicitly set the version but i guess it amounts to the same thing. If there's no behavioral difference I would suggest removing this just to be consistent, but if it's useful let's keep it.

Reply:
Thanks @jarekr. I've copied it verbatim from cp-ksql-server. So are you saying ` <version>${project.version}</version>` can be removed?